l10n-pocket/en/pocket/privacy.ftl

```diff
@@ -7,7 +7,7 @@
 pocket-privacy-title = Privacy
 pocket-privacy-policy = Privacy Policy
 pocket-privacy-posted = Posted October 11, 2021
-pocket-privacy-this-notice = This notice describes the Privacy Policy that applies to all information collected or submitted when you install or use the { -brand-name-pocket } application or any of the other proprietary software applications or services we provide, and when you visit our website (all of which are referred to collectively herein as the "{ -brand-name-pocket-technologies }"). The { -brand-name-pocket-technologies } are owned and licensed by { -brand-name-mozilla-corporation } (referred to herein as "we," "us," "our," "{ -brand-name-mozilla }"), and are operated by { -brand-name-mozilla }’s business unit known as "{ -brand-name-pocket }," which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, { -brand-name-windows }, { -brand-name-mac-short }, { -brand-name-iphone }, { -brand-name-ipad }, { -brand-name-kindle-fire }, { -brand-name-android }, { -brand-name-blackberry }, { -brand-name-windows } phone, { -brand-name-chrome }, { -brand-name-safari }, { -brand-name-firefox } and other mobile devices and computers.
+pocket-privacy-this-notice = This notice describes the Privacy Policy that applies to all information collected or submitted when you install or use the { -brand-name-pocket } application or any of the other proprietary software applications or services we provide, and when you visit our website (all of which are referred to collectively herein as the “{ -brand-name-pocket-technologies }”). The { -brand-name-pocket-technologies } are owned and licensed by { -brand-name-mozilla-corporation } (referred to herein as “we,” “us,” “our,” “{ -brand-name-mozilla }”), and are operated by { -brand-name-mozilla }’s business unit known as “{ -brand-name-pocket },” which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, { -brand-name-windows }, { -brand-name-mac-short }, { -brand-name-iphone }, { -brand-name-ipad }, { -brand-name-kindle-fire }, { -brand-name-android }, { -brand-name-blackberry }, { -brand-name-windows } phone, { -brand-name-chrome }, { -brand-name-safari }, { -brand-name-firefox } and other mobile devices and computers.
```

Comment:
Nit: Not a regression in this diff, but with

> “we,” “us,” “our,” “{ -brand-name-mozilla }”)

having the commas _inside_ the quotes seems incorrect to me, even if it's what was in the source text from legal

Reply:
(It also happens in the TOS strings below)